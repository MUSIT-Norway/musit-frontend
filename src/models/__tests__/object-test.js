import MusitObject from '../object';

describe('Object model', () => {
  it('should work', () => {
    const obj = {
      id: 1,
      term: 'Fugl',
      museumNo: 'Kask123',
      subNo: '1',
      mainObjectId: 1
    };
    expect(MusitObject.getObjectDescription(obj)).toEqual('Kask123 - 1 - Fugl');
    expect(MusitObject.isMainObject(obj)).toBe(true);
  });
});
