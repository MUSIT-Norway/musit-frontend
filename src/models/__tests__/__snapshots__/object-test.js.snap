exports[`MusitObject model should generate empty string on missing props 1`] = `""`;

exports[`MusitObject model should generate proper object description 1`] = `"AK23 - 1 - Rare saker"`;
